_includes/package_links.html

```diff
@@ -13,9 +13,50 @@
 {% endif %}
 
 <div class="list-group list-group-sm {%if horizontal%}list-group-horizontal list-group-justified{%endif%}">
-  <a class="{{list_group_class}}" target="_blank" href="{{ package.data.docs_uri }}" {% unless package.snapshot.documented %}disabled{% endunless %}><span style="margin-right: 5px;" class="glyphicon glyphicon-file"></span>{%if hide_link_labels%}<br>{%endif%} API Docs</a>
-  <a class="{{list_group_class}}" target="_blank" href="{{ package.data.browse_uri }}"><span class="glyphicon glyphicon-folder-open" style="margin-right:5px;"></span>{%if hide_link_labels%}<br>{%endif%} Browse Code</a>
-  <a class="{{list_group_class}} {% unless package.data.wiki.exists %}inactive{% endunless %}" target="_blank" href="http://wiki.ros.org/{{page.package_name}}?distro={{distro}}"><span style="margin-right: 5px;" class="glyphicon glyphicon-info-sign"></span>{%if hide_link_labels%}<br>{%endif%} Wiki</a>
-  <a class="{{list_group_class}} {% unless package.data.wiki.exists %}inactive{% endunless %}" target="_blank" href="http://http404error.github.io/roseco/graph.html?id=ros.json&focus={{page.package_name}}&height=1&depth=2&tred=standard&metagroup=false&colorby=Health&direction=LR" {% unless package.snapshot.documented %}disabled{% endunless %}><span style="margin-right: 5px;" class="glyphicon glyphicon-tree-deciduous"></span>{%if hide_link_labels%}<br>{%endif%}</a>
+  <a class="{{list_group_class}}"
+     target="_blank"
+     href="{{ package.data.docs_uri }}"
+     title="View API documentation on docs.ros.org"
+     {% unless package.snapshot.documented %}disabled{% endunless %}>
+       <span style="margin-right: 5px;" class="glyphicon glyphicon-file"></span>
+       {%if hide_link_labels%}<br>{%endif%}
+       API Docs
+  </a>
+  <a class="{{list_group_class}}"
+     target="_blank"
+     href="{{ package.data.browse_uri }}"
+     title="View source code on repository">
+       <span class="glyphicon glyphicon-folder-open" style="margin-right:5px;"></span>
+       {%if hide_link_labels%}<br>{%endif%}
+       Browse Code
+  </a>
+  {% if page.available_distros["noetic"]
```

Comment:
This list is available in the site variables. https://github.com/ros-infrastructure/rosindex/blob/ros2/_config.yml#L94

I think you can get the list as `site.ros_distros + site.old_ros_distros` instead of hard coding them here. 

Does liquid handle key errors appropriately on a dict for an if? It's usually a bad idea to test for keys in an if statement. I usually want to test 'foo' in page.available_distros  or something more like `if set(site.old_ros_distros + site.ros_distros) & set(page.available_distros.keys())`

Reply:
Thanks. I switched to using `ros_distro` and `old_ros_distro` instead.

As for handling missing keys, using liquid `if` to test for a keys was used in other parts of the code. If that's a problem I can search for some different approaches.

Reply:
Thanks, the missed keys I was thinking python. But this is ruby so it's slightly different and if the pattern works elsewhere hopefully it works here too.